webapp/src/containers/components/modals/deleteConfigModal.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from 'react';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import './styles.css';
+
+interface Props {
+    visible: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+function DeleteConfigModal(props: Props) {
+    const [show, setShow] = useState(false);
+
+    useEffect(() => {
+        setShow(props.visible);
+    }, [props.visible]);
+
+    const handleClose = () => {
+        setShow(false);
+        props.setVisibility(false);
+    };
+    return (
+        <>
```

Comment:
why you are using this?

Reply:
removed the <></> since we have only one component in file

Reply:
didn't get what you are suggesting on lines 46-49

Reply:
You need to add an empty line at the end of every file, see the red icon below it means there is no new line
![image](https://github.com/Brightscout/mattermost-plugin-welcomebot/assets/72438220/60c338a8-b627-4db0-b64a-651e066ee831)

Reply:
ok